src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -2047,6 +2047,8 @@ internal ImmutableArray<BoundNode> GetBoundNodes(CSharpSyntaxNode node)
             }
             Debug.Assert(node == GetBindableSyntaxNode(node));
 
+            // Note: This nullability analysis can be distracting when debuggging. This can be disabled with a feature flag in parse options:
```

Comment:
```suggestion
            // Note: This nullability analysis can be distracting when debugging. This can be disabled with a feature flag in parse options:
```